project/profiles/capacity4more/modules/c4m/user/c4m_user/actions/c4m_user_contact_multiple.action.inc

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * @file
+ * Callback & forms related to the user contact multiple.
+ */
+
+/**
+ * Implements hook_action_info().
+ *
+ * A custom action to contact a multiple users with a mail action.
+ */
+function c4m_user_contact_multiple_action_info() {
+  $group = c4m_og_current_group();
+  // If not within OG context, don't declare action.
+  if (!$group) {
+    return array();
+  }
+  // If not Site Admin, Group Admin or Group Owner, don't declare action.
+  if (!_c4m_features_og_members_is_power_user($group)) {
+    return array();
+  }
+  // Site Admin, Group Admin or Group Owner => declare action.
+  return array(
+    'c4m_user_contact_multiple_action' => array(
+      'type' => 'user',
+      'label' => t('Contact'),
+      'configurable' => TRUE,
+      'triggers' => array('any'),
+    ),
+  );
+}
+
+/**
+ * Create 'Contact Multiple' form.
```

Comment:
It looks like this is an implementation of _hook_form_, we should declare this on the docs (see other hooks).